docs/releases/2.2.1.txt

```diff
@@ -33,3 +33,6 @@ Bugfixes
 * Reverted an optimization in Django 2.2 (:ticket:`29725`) that caused the
   inconsistent behavior of ``count()`` and ``exists()`` on a reverse
   many-to-many relationship with a custom manager (:ticket:`30325`).
+
+* Fixed crash when ``ordering`` or ``aggregating`` over a deep
```

Comment:
Both `ordering` and `aggregating` shouldn't be in backtips and _deep_ should probably be replaced by _nested_.